packages/@react-spectrum/dropzone/stories/DropZone.stories.tsx

```diff
@@ -166,23 +173,23 @@ function Example(props) {
 
 function DropZoneWithDraggable(props) {
   let [isFilled, setIsFilled] = useState(false);
-  let [filledSrc, setFilledSrc] = useState(null);
+  let [filledSrc, setFilledSrc] = useState<string | null>(null);
 
   return (
     <>
       <Draggable />
-      <DropZone 
+      <DropZone
         {...props}
         isFilled={isFilled}
-        onDrop={async (e) => {
-          let items = await Promise.all(e.items.filter((item) => item.kind === 'text' && item.types.has('text/plain')).map((item: TextDropItem) => item.getText('text/plain')));
+        onDrop={async (e: DropEvent) => {
+          let items = await Promise.all(e.items.filter((item) => item.kind === 'text' && item.types.has('text/plain')).map((item) => (item as TextDropItem).getText('text/plain')));
```

Comment:
This came from docs, I think this [example](https://react-spectrum.adobe.com/react-aria/useDrop.html#example). If this is correct we should update those too.

Reply:
this is fine, for other examples of ways to solve this, see https://www.skovy.dev/blog/typescript-filter-array-with-type-guard?seed=64zeg1

Reply:
Thanks for sharing. I tried the second suggestion and `let items = await Promise.all(e.items.filter((item): item is TextDropItem => item.kind === 'text' && item.types.has('text/plain')).map((item) => (item).getText('text/plain')));` works and allows us not to set the event type. I couldn't find us using 'is' in any /src/* files in the project. This is very good to know!